src/main/resources/application.conf

```diff
@@ -158,6 +158,12 @@ vsys {
     #        ]
     #      }
     #    }
+
+    # State settings
+    state {
+      # turn on/off the state of transactions grouped by address and transaction type 
```

Comment:
is it true that change the setting from `off` to `on`, user need to rebuild the state DB?

if yes, we need extra comment here to remind the change.

Reply:
Added more comments above the block 'state'